src/Pelagos/Entity/ResearchGroup.php

```diff
@@ -652,13 +661,49 @@ public function setEmailAddress($emailAddress)
      *
      * @access public
      *
-     * @return string Containing emailADdress.
+     * @return string Containing emailAddress.
      */
     public function getEmailAddress()
     {
         return $this->emailAddress;
     }
 
+    /**
+     * Method to lock Research Group.
+     *
+     * @access public
+     *
+     * @return void
+     */
+    public function lockResearchGroup()
+    {
+        $this->locked = true;
+    }
+
+    /**
+     * Method to unlock Research Group.
+     *
+     * @access public
+     *
+     * @return void
+     */
+    public function unlockResearchGroup()
+    {
+        $this->locked = false;
+    }
+
+    /**
+     * Method to check if Research Group is locked.
+     *
+     * @access public
+     *
+     * @return boolean Set to true if this Research Group is locked, false otherwise.
+     */
+    public function isResearchGroupLocked()
```

Comment:
Is that extra "researchGroup" on these functions really needed? This is how it might look:
`if $researchGroup->isResearchGroupLocked()`
This is a long standing debate, but I think just isLocked is sufficient.